### ISSUE TITLE
Accessibility - Search Screen has Color Contrast Issues

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Closure-App#4]
There are a few different color contrast issues with the select a city screen.

The red text has a color contrast ratio of 3.63:1. While the light blue toggle sections border and highlighted button only has a color contrast ratio of 4.02:1. Plus, the search box border has a color contrast ratio of 1.43:1 and the text int he box is only 4.33:1.

To pass the color contrast ratio needs to be 4.5:1 or higher.

The right-side navigation barely has enough color contrast to pass. Maybe we could make that text black or #3d3e3d (dark gray) to make it easier to read because it's too small.

You can use TPG's color contrast analyzer - https://developer.paciellogroup.com/resources/contrastanalyser/ I have used the Windows version.

See screenshot below.

![search screen has color contrast issues](https://user-images.githubusercontent.com/10423487/34798323-9d654306-f629-11e7-848d-8d13d022ad8b.PNG)

**Comment by georgepliu** [comment]
All 'orange' colored items have been changed to black.

This issue has been resolved and is reflected in the current build #11

**Comment by jfc3-dol** [comment]
The magnifying glass icon and the text "Done" now have enough color contrast to pass. but the light blue text and background with white only has a color contrast ration of 4.01:1 and needs to be 4.5:1 or higher to pass.

If you make the light blue a bit darker say #036FE8 it will pass with color contrast ratio of 4.73:1.

See screenshot below.

![closure notification - light blue and white have poor color contrast](https://user-images.githubusercontent.com/10423487/36499300-d5598280-170e-11e8-9b11-3497b16db05e.png)
